### PR TITLE
Init for SpringCloud.

### PR DESCRIPTION
**Pull request by wyunchi-ms**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: tools/CreateMappings_rules.json [hunk 1]
```diff
@@ -1,190 +1,580 @@
 [
```
Comment (dingmeng-xue) [inline tools/CreateMappings_rules.json:1]: Format should not be changed.

File: src/SpringCloud/Az.SpringCloud.psd1 [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        LicenseUri = 'https://aka.ms/azps-license'
+
+        # A URL to the main website for this project.
+        ProjectUri = 'https://github.com/Azure/azure-powershell'
+
+        # A URL to an icon representing this module.
+        # IconUri = ''
+
+        # ReleaseNotes of this module
+        # ReleaseNotes = ''
```
Comment (dingmeng-xue) [inline src/SpringCloud/Az.SpringCloud.psd1:118]: Release notes should not be empty.

**Review by dingmeng-xue — CHANGES_REQUESTED** [review]
https://github.com/Azure/azure-powershell/blob/ddaf85533f906be531a00d1361b19b8a3aa302af/tools/GenerateHelp.ps1#L85

This file needs to be changed too.

**Review by dingmeng-xue — APPROVED** [review]

**Pull request merged**